perfaware/sim86/shared/Sim8086.cpp

```diff
@@ -4,13 +4,11 @@
 #include <string>
 #include <sstream>
 #include "sim86_shared.h"
+#include <assert.h>
```

Comment:
There is a common convention about the order in which headers are included. I have often seen people include standard headers before their own headers. It's just good to know that this is something people do, and so I would move assert above sim_shared

Reply:
It helps to avoid situations where a standard header redefines something you have defined in one of your own headers, because it was included later

Reply:
Understood, thanks!
c424a077bd42dd15109828bc5f5e15800501f310